src/MenuActiveEffects.h

```diff
@@ -39,15 +39,19 @@ class EffectIcon {
 	int type;
 	int current;
 	int max;
+	int stacks;
 	Rect pos;
 	Rect overlay;
 	std::string name;
+	WidgetLabel* stacksLabel;
 
 	EffectIcon()
 		: icon(-1)
 		, type(0)
 		, current(0)
 		, max(0)
+		, stacks(0)
+		, stacksLabel(nullptr)
```

Comment:
As per [this](https://github.com/clintbellanger/flare-engine/wiki/Code-Conventions#no-c11--c0x), I ask that you just use NULL or 0 here, since nullptr is C++11.